### DISCUSSION TITLE
How to add different versions of postgresql native clients?

### DISCUSSION THREAD

**Discussion by victormgomes** [dbeaver/dbeaver#18879] [Q&A]
I have dbeaver community flatpak installed on my fedora 37 and i'm trying to backup a postgresql database using it.
But the postgresql server is version 13 and the native client located at:
/var/lib/flatpak/runtime/io.dbeaver.DBeaverCommunity.Client.pgsql/x86_64/stable/active/files
is version 12.
Can someone help me to add a version 13 client?
thanks!

**Comment by LonwoLonwo** [comment]
Hello. You can do it in the Connections Settings -> Local client -> Browse

![2023-01-30 20_19_18-DBeaver Ultimate 22 3 3 - 11 24 324FFd](https://user-images.githubusercontent.com/45152336/215548522-f56552a6-bee9-48fd-b74b-50ddc552a337.png)

And then click on the "Add home" button and specify the path to your local database folder

![2023-01-30 20_19_45-DBeaver Ultimate 22 3 3 - 11 24 324FFd](https://user-images.githubusercontent.com/45152336/215548751-011ae73b-06b2-41fe-b192-e1758cd63531.png)

**Reply by victormgomes** [reply]
Hi i bypass the situation by downloading the version 13 binaries  from the repositories of postgresql and put them in a directory and them on add home option.
But i dont know where i can get all these postgresql clients as packages that you have in the last image.
Thanks!
![Captura de tela de 2023-01-30 14-31-10](https://user-images.githubusercontent.com/20375480/215551059-7cbc4ee9-117f-493b-9beb-5ca960bf0bc5.png)
![Captura de tela de 2023-01-30 14-32-06](https://user-images.githubusercontent.com/20375480/215551078-73df426c-62b9-4537-bac7-352ccfd34855.png)

**Reply by LonwoLonwo** [reply]
I downloaded PostgreSQL from the official site: https://www.postgresql.org/download/

**Reply by rausub** [reply]
> But i dont know where i can get all these postgresql clients as packages that you have in the last image.

Yeah did you manually download those ?
